tests/unit/mixins/test_mixin_methods.py

```diff
@@ -24,108 +24,127 @@ class FakeManager(base.RESTManager):
     _obj_cls = FakeObject
 
 
+@responses.activate
 def test_get_mixin(gl):
     class M(GetMixin, FakeManager):
         pass
 
-    @urlmatch(scheme="http", netloc="localhost", path="/api/v4/tests/42", method="get")
-    def resp_cont(url, request):
-        headers = {"Content-Type": "application/json"}
-        content = '{"id": 42, "foo": "bar"}'
-        return response(200, content, headers, None, 5, request)
+    url = "http://localhost/api/v4/tests/42"
+    responses.add(
+        method=responses.GET,
+        url=url,
+        json={"id": 42, "foo": "bar"},
+        status=200,
+        match_querystring=True,
+    )
 
-    with HTTMock(resp_cont):
-        mgr = M(gl)
-        obj = mgr.get(42)
-        assert isinstance(obj, FakeObject)
-        assert obj.foo == "bar"
-        assert obj.id == 42
+    mgr = M(gl)
+    obj = mgr.get(42)
+    assert isinstance(obj, FakeObject)
+    assert obj.foo == "bar"
+    assert obj.id == 42
+    assert responses.assert_call_count(url, 1) is True
```

Comment:
As we aren't sharing any fixtures here, was the assert for 1 call intended for all methods or mostly for the one with 2 calls?

Maybe we could split `test_list_mixin` into `test_list_mixin` and `test_list_mixin_as_generator` then to really only test a single scenario per case, and use responses' own assert.

Reply:
we can do that in a follow-up, we still have some httmock anyway.